app/containers/activity/screen/canvas/CameraInput.js

```diff
@@ -23,100 +20,123 @@ const styles=StyleSheet.create({
   footerText: {
     fontSize: 20,
     fontWeight: '300',
+  }, buttonText: {
+    fontSize: 24,
+    fontWeight: '300'
+  },
+  takeButton: {
+    borderRadius: 12,
+    width: '100%',
+    height: 360,
+    alignContent: 'center',
+    justifyContent: 'center',
+    alignItems: 'center',
+    borderWidth: 4,
+    borderColor: '#d10000',
+    backgroundColor: '#ffdddd'
+  },
+  videoConfirmed: {
+    borderRadius: 12,
+    width: '100%',
+    height: 360,
+    alignContent: 'center',
+    justifyContent: 'center',
+    alignItems: 'center',
+    borderWidth: 4,
+    borderColor: '#00a30a',
+    backgroundColor: '#99ff9f'
   },
-  flipButton: {
-    position: 'absolute',
-    bottom: 8,
-    left: 8,
+  chooseButton: {
+    borderRadius: 12,
+    width: '100%',
+    height: 116,
+    alignContent: 'center',
+    justifyContent: 'center',
+    alignItems: 'center',
+    borderWidth: 4,
+    borderColor: '#000000',
+    backgroundColor: '#dbdbdb',
+    marginTop: 8,
   },
-  pickButton: {
-    position: 'absolute',
-    bottom: 8,
-    right: 8,
+  redIcon: {
+    color: '#d10000',
+    fontSize: 60,
   },
+  greenIcon: {
+    color: '#00a30a',
+    fontSize: 60,
+  }
 });
```

Comment:
Ideally we should pull the colors from a central place to encourage standardization and reuse.

Simplest way to do this would be to add the colors to a theme file in `app/themes` and import from there.

In the future we will need to take it a step further and make the themes switchable. There are a few options for this but we should probably discuss that as a team.